apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Program.cs

```diff
@@ -53,12 +51,15 @@ private static int Main(string[] args)
             try
             {
                 XDocument document = XDocument.Load(args[0]);
-                var code = File.ReadLines(args[1]);
-                foreach (var methodElement in document.Root.Elements("Method"))
-                {
-                    ApiMethod method = new ApiMethod(methodElement);
-                    code = ModifyCode(method, code);
-                }
+                var methods = document.Root.Elements("Method").Select(element => new ApiMethod(element));
+                var methodCode = methods.SelectMany(method => method.GenerateCode()).ToList();
```

Comment:
super nit: methodCode => allMethodsCode or any other plural name?
(I saw `ToList()` and inferred that this variable holds code for all the methods and then got confused if it's just one method code based on the variable name)

Reply:
Sure, happy to do so.

Reply:
Done.